app/controllers/concerns/finder_top_result_ab_testable.rb

```diff
@@ -0,0 +1,28 @@
+module FinderTopResultAbTestable
+  CUSTOM_DIMENSION = 69
+
+  def self.included(base)
+    base.helper_method(
+      :finder_top_result_test_variant,
+      :show_top_result?
+    )
+    base.after_action :set_test_response_header
+  end
+
+  def finder_top_result_test
+    @finder_top_result_variant ||= GovukAbTesting::AbTest.new(
```

Comment:
Linter is upset because the instance variable name is not the same as the method name.